library/core/src/char/methods.rs

```diff
@@ -597,6 +597,8 @@ impl char {
     /// Returns the number of 16-bit code units this `char` would need if
     /// encoded in UTF-16.
     ///
+    /// That number of code points is always 1 or 2.
```

Comment:
fix: I think "code points" is the wrong term to use here, since that primarily means [the 21-bit number](http://www.unicode.org/glossary/#code_point).  The previous paragraph intentionally used [code unit](http://www.unicode.org/glossary/#code_unit), which is the correct term here.

So consider a tweak like
```suggestion
    /// That number of code units is always either 1 or 2, for unicode scalar values in
    /// the [basic multilingual plane](http://www.unicode.org/glossary/#basic_multilingual_plane)
    /// or [supplementary planes](http://www.unicode.org/glossary/#supplementary_planes)
    /// respectively.
```